src/config/types.ts

```diff
@@ -112,7 +112,7 @@ export interface BallotContextInterface {
 }
 
 // Smart Card Content
-export type CardDataTypes = 'voter' | 'admin'
+export type CardDataTypes = 'voter' | 'pollworker' | 'clerk'
```

Comment:
nit for later: "CardData" refers to short value only, but one could easily assume that it is all the card data.